QuantConnect.IBAutomater/IBAutomater.cs

```diff
@@ -899,6 +900,33 @@ private string GetIbGatewayVersionPath()
             return ibGatewayVersionPath;
         }
 
+        private string GetIbGatewayIniFile()
+        {
+            return Path.Combine(IsWindows ? GetIbGatewayVersionPath() : _ibDirectory, "jts.ini");
+        }
+
+        private void UpdateIbGatewayIniFile()
+        {
+            var ibGatewayIniFile = GetIbGatewayIniFile();
+            OutputDataReceived?.Invoke(this, new OutputDataReceivedEventArgs($"Updating IBGateway ini file: {ibGatewayIniFile}"));
+
+            // The "Use SSL" checkbox in the IBGateway UX has inconsistent behavior,
+            // so we preset the "UseSSL=true" value in jts.ini
+            if (File.Exists(ibGatewayIniFile))
+            {
+                var iniText = File.ReadAllText(ibGatewayIniFile);
+                if (iniText.Contains("UseSSL=false"))
+                {
+                    iniText = iniText.Replace("UseSSL=false", "UseSSL=true");
+                    File.WriteAllText(ibGatewayIniFile, iniText);
+                }
```

Comment:
What if the file exists but doesn't have the `UseSSL`, we should probably add it? We are kind of supposing that if `UseSSL=false` isn't found it means it's true already

Reply:
I tested fresh IBGateway installations on both Windows and Linux and the jts.ini file is never created by the setup program.
It is only created after a successful login (defaulting to `UseSSL=false`).
I think the case you mention would only possible if the file was edited manually and removing the line.